Vagrantfile

```diff
@@ -434,20 +434,24 @@ Vagrant.configure("2") do |config|
         $mountNfsShare = ''
         if(vagrant_provider == 'virtualbox')
             $mountNfsShare = <<-'SCRIPT'
-            if ! "$(mount | grep /vagrant)" ; then
+            if ! mount | grep -qs /vagrant ; then
```

Comment:
I think we can put set -e from the very beginning 
now we know that what is after the "if" is correct, but if we had this "set -e" in place from the beginning it would have spotted my error... so it can be seen as a forward protection for future mistakes 
what do you think ?

Reply:
before the if you mean? Yes